validator/graffiti/parse_graffiti.go

```diff
@@ -1,13 +1,21 @@
 package graffiti
 
 import (
+	"encoding/hex"
 	"io/ioutil"
+	"strings"
 
 	types "github.com/prysmaticlabs/eth2-types"
 	"github.com/prysmaticlabs/prysm/shared/hashutil"
+	"github.com/sirupsen/logrus"
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	hexGraffitiPrefixKey = "hex"
```

Comment:
nit: could just be `hexGraffitiPrefix` (it's either a key or a prefix, no need for both)